src/chrome/content/rules/administrator.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="administrator.de">
+<target host="*.administrator.de" />
```

Comment:
I'm fine with using a rule that rewrites all subdomains when there is only www. It has the big advantage that it makes it impossible for a MITM to make you load nonexistent.example.com over HTTP and potentially leak cookies if they are scoped to *.example.com.